hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/BPOfferService.java

```diff
@@ -811,9 +811,10 @@ assert getBlockPoolId().equals(bp) :
    */
   private boolean processCommandFromStandby(DatanodeCommand cmd,
       BPServiceActor actor) throws IOException {
+    String who = ", NameNode at " + actor.getNNSocketAddress();
     switch(cmd.getAction()) {
     case DatanodeProtocol.DNA_ACCESSKEYUPDATE:
-      LOG.info("DatanodeCommand action from standby: DNA_ACCESSKEYUPDATE");
+      LOG.info("DatanodeCommand action from standby: DNA_ACCESSKEYUPDATE" + who);
```

Comment:
It might be better to use a placeholder like:
`LOG.info("DatanodeCommand action from standby NN {}: DNA_ACCESSKEYUPDATE, actor.getNNSocketAddress());`

Several other places can be used in a similar manner.

Reply:
Thanks @tomscut for the comment.
I will update it later.